index.html

```diff
@@ -729,6 +729,11 @@ <h3 id="configuration">Configuration</h3>
         <td><i>null</i></td>
         <td>A custom CSS class for the modal.</td>
       </tr>
+      <tr>
+        <td><b>customContainerClass</b></td>
+        <td><i>null</i></td>
```

Comment:
It's `''`, not `null`. Also could you please make the same fix (`null` -> `''`) for line 729

Reply:
Should i also change line 809, 814, 888 and 928? They are all custom class

Reply:
Done! I also created https://github.com/sweetalert2/sweetalert2/pull/1352